client/jetpack-cloud/sections/partner-portal/payment-method-delete-dialog/index.tsx

```diff
@@ -27,6 +29,8 @@ const PaymentMethodDeleteDialog: FunctionComponent< Props > = ( {
 		digits: paymentMethod?.card.last4,
 	} );
 
+	const { isFetching: isFetchingRecentPaymentMethods } = useRecentPaymentMethodsQuery();
```

Comment:
I realized we can optimize this a little bit - let's pass an options object to the custom hook that is merged with the underlying useQuery options. This way we can pass `enabled: paymentMethod.is_default` so we don't fetch and wait in cases where the deleted payment method is not the default one.

Reply:
Nice suggestion! Updated in e121f57fab0a56c5653bf4d67a99a44958db2a81